python/src/trezorlib/transport/__init__.py

```diff
@@ -96,21 +105,23 @@ def find_by_path(cls, path: str, prefix_search: bool = False) -> "Transport":
         raise TransportException(f"{cls.PATH_PREFIX} device not found: {path}")
 
 
-def all_transports() -> Iterable[Type[Transport]]:
+def all_transports() -> Iterable[Type["TransportType"]]:
```

Comment:
why change the signature from `Type[Transport]` to `Type[TransportType]`?

AFAICT `Type[Transport]` is more correct -- we're guaranteeing that the output implements the `Transport` interface but not more.

Reply:
With `Type[Transport]` it was complaining that some method (I think `find_debug()`) is missing on `Transport`